### ISSUE TITLE
Pinned replies aren't shown at the top of profiles

### ISSUE THREAD

**Issue by ShredderWalnut** [mastodon/mastodon-ios#332]
When pinning toots to my profiles, the pinned toots aren't shown at the top if it is a reply I wrote. The web UI shows the pinned roots correctly, though.

**Comment by FediVideos** [comment]
Yes, this is a significant problem on some accounts because the pinned toots give important context to people unfamiliar with the account. Some accounts assume people will see those pinned toots.

**Comment by ShredderWalnut** [comment]
> there are no pinned toots of any kind on the latest version

Does this imply that in previous versions, pinned toots that aren't replies are displayed correctly in the app? If so, should I maybe edit the issue title from "Pinned replies…" to "Pinned toots…"?
